hack/setup/infra/manage.prometheus-adapter-helm.sh

```
@@ -0,0 +1,108 @@
+#
+# Environment variables for custom Helm values:
+#   PROMETHEUS_ADAPTER_EXTRA_ARGS - Additional helm install arguments
+#   PROMETHEUS_URL - Prometheus server URL (default: http://prometheus-kube-prometheus-prometheus.monitoring:9090)
+#
+# Configures the adapter to expose the wva_desired_replicas metric
+# via the Kubernetes external metrics API for HPA consumption.
+
+# INIT
+SCRIPT_DIR="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)"
+
+source "${SCRIPT_DIR}/../common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../common.sh was not specified as input (see shellcheck -x).

(SC1091)